modules/nf-core/shapeit5/switch/tests/main.nf.test

```diff
@@ -0,0 +1,104 @@
+nextflow_process {
+
+    name "Test Process SHAPEIT5_SWITCH"
+    script "../main.nf"
+    process "SHAPEIT5_SWITCH"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "shapeit5"
+    tag "shapeit5/switch"
+    tag "shapeit5/phasecommon"
+    tag "bcftools/index"
+
+    test("homo sapiens - vcf, scaffold, []") {
+        config "./nextflow.config"
+        setup {
+            run("SHAPEIT5_PHASECOMMON") {
+                script "../../../shapeit5/phasecommon"
+                params {
+                    shapeit5_phasecommon_args = "--filter-maf 0.001"
+                }
+                process {
+                    """
+                    input[0] = Channel.of([
+                        [ id:'scaffold', single_end:false ], // meta map
+                        file(params.modules_testdata_base_path + "delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf", checkIfExists: true),
+                        file(params.modules_testdata_base_path + "delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf.csi", checkIfExists: true),
+                        [],
+                        "chr21",
+                    ])
+                    input[1] = Channel.of([[],[],[]])
+                    input[2] = Channel.of([[],[],[]])
+                    input[3] = Channel.of([[],[]])
+                    """
+                }
+            }
+            run("BCFTOOLS_INDEX") {
+                script "../../../bcftools/index"
+                process {
+                    """
+                    input[0] = SHAPEIT5_PHASECOMMON.out.phased_variant
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = SHAPEIT5_PHASECOMMON.out.phased_variant
+                    .join(BCFTOOLS_INDEX.out.csi)
+                    .combine(Channel.of("chr21"))
+                    .combine(Channel.of([[]]))
+                input[1] = Channel.of([[ id:'truth_panel'],
+                    file(params.modules_testdata_base_path + "delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf",checkIfExists:true),
+                    file(params.modules_testdata_base_path + "delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf.csi",checkIfExists:true)
```

Comment:
Can we please do something with this dataset on the test-data repository, rather than leave it in the delete_me folder? You've got multiple modules here using it.

Reply:
You're totally right.
I'll discuss it in the dedicated slack channel and update this pull request when done !

Reply:
Those files are also present on the `phaseimpute` branch of the test-datasets repository.
Should I use these files ?

Reply:
sure that also works